controllers/projectclaim_controller.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controllers
```

Comment:
doesn't this mean that the 2 controllers can share data?
they are supposed to be separate AFAICT
same goes to the tests

Reply:
The same thought crossed my head too, seeing the two controllers were separate pre-migration.
Operator SDK by default generated all the controllers under the same folder and I kept it that way and wanted the reviewers to make a call on this.
Let me try separating them now...

Reply:
Separation done.